requirements.txt

```diff
@@ -3,7 +3,7 @@ PyYAML
 requests>=2.0
 requests-mock>=0.7,<0.8
 django-crispy-forms>=1.6,<1.7
-django-selectable>=0.9.0,<1.0
+git+https://github.com/mlavin/django-selectable.git@master#egg=django-selectable
```

Comment:
I will proceed to pin the version correct to the latest commit.
```
git+https://github.com/mlavin/django-selectable.git@<commit_id>#egg=django-selectable
```

Does that solve the security issue?